Cesium3DTilesSelection/src/RasterOverlayUpsampler.cpp

```diff
@@ -57,14 +57,19 @@ RasterOverlayUpsampler::loadTileContent(const TileLoadInput& loadInput) {
           parentProjections.begin(),
           parentProjections.end(),
           projection);
-      index = int32_t(it - parentProjections.begin());
+      index = static_cast<size_t>(it - parentProjections.begin());
       break;
     }
   }
 
+  const CesiumGeospatial::Projection& projection = parentProjections.at(index);
+  const CesiumGeospatial::Ellipsoid& ellipsoid =
+      getProjectionEllipsoid(projection);
+
   const CesiumGltf::Model& parentModel = pParentRenderContent->getModel();
   return loadInput.asyncSystem.runInWorkerThread(
       [&parentModel,
+       &ellipsoid,
```

Comment:
This one _might_ be safe to capture by reference, but unless you're very sure it's better to just capture it by value.